app/src/main/java/com/arygm/quickfix/ui/profile/ProfileScreen.kt

```diff
@@ -34,269 +41,265 @@ import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.arygm.quickfix.R
 import com.arygm.quickfix.model.account.LoggedInAccountViewModel
 import com.arygm.quickfix.ui.navigation.NavigationActions
 import com.arygm.quickfix.ui.navigation.Screen
 import com.arygm.quickfix.ui.navigation.TopLevelDestinations
-import com.google.firebase.Firebase
-import com.google.firebase.auth.auth
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun ProfileScreen(
     navigationActions: NavigationActions,
     loggedInAccountViewModel: LoggedInAccountViewModel,
-    navigationActionsRoot: NavigationActions
+    navigationActionsRoot: NavigationActions,
 ) {
-  val loggedInProfile by loggedInAccountViewModel.loggedInAccount.collectAsState()
-  // List of options handled by the profile screen
-  val options =
-      listOf(
-          OptionItem("Settings", IconType.Vector(Icons.Outlined.Settings)) {},
-          OptionItem("Activity", IconType.Resource(R.drawable.dashboardvector)) {},
-          OptionItem("Set up your business account", IconType.Resource(R.drawable.workvector)) {
-            navigationActions.navigateTo(Screen.TO_WORKER)
-          },
-          OptionItem("Account configuration", IconType.Resource(R.drawable.accountsettingsvector)) {
-            navigationActions.navigateTo(Screen.ACCOUNT_CONFIGURATION)
-          },
-          OptionItem("Workers network", IconType.Vector(Icons.Outlined.Phone)) {},
-          OptionItem("Legal", IconType.Vector(Icons.Outlined.Info)) {})
+  val loggedInAccount by loggedInAccountViewModel.loggedInAccount.collectAsState()
+  val loggedInProfile by loggedInAccountViewModel.userProfile.collectAsState()
+  val displayName =
+      loggedInAccount?.let { capitalizeName(it.firstName, it.lastName) } ?: "Loading..."
+  val email = loggedInAccount?.email ?: "Loading..."
+  val wallet = loggedInProfile?.wallet ?: "Loading..."
```

Comment:
`LoggedInAccountViewModel` is deprecated. This screen will need to be refactored **as soon as** the offline mode PR is merged.

Reply:
I think we can merge this PR for now but I'll make sure to open a new PR as soon as the offline mode PR is merged